### ISSUE TITLE
Suggestion: Add a new notification setting for pulling RSS feeds

### ISSUE THREAD

**Issue by RoY7x** [jonas747/yagpdb#469]
I think title is pretty easy to get. Something to be able to pull RSS feeds like from news sites, or other corners of the internet. I think there is already a couple of Discord bots that do it already, however adding it to YAGPDB might just add further support to an already impressive list of things it can do. It was something that came to mind after finding use with the Reddit notifications.

**Comment by raulcraveiro** [comment]
I really need this and would love to see it on YAGPDB ❤

**Comment by XDelta** [comment]
Custom RSS feeds would be a good addition. Would be good to have placeholders/template data for structuring the messages on a per feed basis as well.

**Comment by paride5745** [comment]
I'd love this feature as well, at the moment I am using MonitoRSS but I would prefer to use YAGPDB for these kind of feeds as well.

**Comment by ajf8729** [comment]
Agreed, this would be a great feature, also currently looking at MonitoRSS.
